File: submissions/sm_026_rohit-kumar/week_03/day_2/assignment/taxCalculator.js
var income = 500000
var savings = 50000
tax = 0;


if (income >= 250000 && income < 500000){
    rebate = savings * 0.50;
    rebate = (rebate > 50000) ? 50000 : rebate
    tax = (income - rebate) * 0.10

} else if(income >= 500000 && income < 1000000){
    rebate = savings * 0.30
    rebate = (rebate > 50000) ? 50000 : rebate
    tax = (income - rebate) * 0.20
} else if(income > 1000000){
    rebate = savings * 0.10
    rebate = (rebate > 50000) ? 50000 : rebate
    tax = (income - rebate) * 0.30
}

console.log("Total payable Tax : ", tax)

File: submissions/sm_026_rohit-kumar/week_13/day_5/session_1/tax_calculator.py
def find_tax(income):
    if income >= 250000 and income < 500000:
        rebate = savings * 0.50
        rebate = 50000 if (rebate > 50000) else rebate

    elif income >= 500000 and income < 1000000:
        rebate = savings * 0.30
        rebate = 50000 if (rebate > 50000) else rebate

    elif income >= 1000000:
        rebate = savings * 0.10
        rebate = 50000 if (rebate > 50000) else rebate
    else:
        print('No tax')
        return

    taxable_income = income - rebate

    if taxable_income >= 250000 and taxable_income < 500000:
        tax = taxable_income * 0.10

    elif taxable_income >= 500000 and taxable_income < 1000000:
        tax = taxable_income * 0.20
    elif taxable_income >= 1000000:
        tax = taxable_income * 0.30

    print("Total payable Tax : ", tax)



income = int(input('Enter you income : '))
savings = int(input('Enter you savings : '))
tax = 0

find_tax(income)

